bird2/Makefile

```diff
@@ -7,12 +7,20 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=bird2
+<<<<<<< HEAD
 PKG_VERSION:=2.13.1
+=======
+PKG_VERSION:=2.15.1
+>>>>>>> a55a0d1 (bird2: bump to version 2.15.1)
```

Comment:
It seems that merge conflict was not properly resolved. Would you mind to take a look, please? :)

Reply:
Sorry for my mistake… It's fixed now